dts/bindings/led/ti,lp5569.yaml

```diff
@@ -3,3 +3,21 @@ description: TI LP5569 LED
 compatible: "ti,lp5569"
 
 include: i2c-device.yaml
+
+properties:
+  enable-gpios:
+    type: phandle-array
+    description: |
+      Enable pin of the lp5569.
+
+  charge-pump-mode:
+    type: int
```

Comment:
Could be an enum

Reply:
I think it's cleaner to just use the bit configuration as it is specified in the datasheet. But yeah, it could be enum/defined values